UnitsNet/UnitDefinitions/AcElectricPotential.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "AcElectricPotential",
```

Comment:
I think rename to `ElectricPotentialAc` and `ElectricPotentialDc` below, so they units and json files get grouped together. Should help discoverability.